codegenerator/cli/templates/static/codegen/src/Time.res

```diff
@@ -14,24 +14,23 @@ let rec retryAsyncWithExponentialBackOff = async (
   | exn =>
     if retryCount < maxRetries {
       let nextRetryCount = retryCount + 1
-      exn
-      ->ErrorHandling.make(
-        ~logger?,
-        ~msg=`Failure. Retrying ${nextRetryCount->Belt.Int.toString}/${maxRetries->Belt.Int.toString} in ${backOffMillis->Belt.Int.toString}ms`,
-      )
-      ->ErrorHandling.log
+      logger->Logging.childWarn({
+        "message": `Failure. Retrying ${nextRetryCount->Belt.Int.toString}/${maxRetries->Belt.Int.toString} in ${backOffMillis->Belt.Int.toString}ms`,
```

Comment:
:pray:

Reply:
```suggestion
        "message": `Retrying query ${nextRetryCount->Belt.Int.toString}/${maxRetries->Belt.Int.toString} in ${backOffMillis->Belt.Int.toString}ms - waiting for correct result.`,
```
I'm not sure how best to update this wording since it is a function that is reused, but I think we will get less error reports with this wording than using the word failure.

Reply:
It's fine, since the function only reused in the places where we query logs